projects/spl-governance/index.js

```diff
@@ -1,22 +1,13 @@
 const { getCache } = require('../helper/http')
-const url = 'https://api.realms.today/stats/tvl'
+const url = 'https://realms-tvl.vercel.app/tvl/latest'
```

Comment:
thanks for the update, but we would like a token breakdown, or even better a way to pull the values on chain

Reply:
> thanks for the update, but we would like a token breakdown, or even better a way to pull the values on chain

The API has been updated to provide token-wise data. Fetching the token amounts for every DAO has a lot of overhead so we are providing it via API but we have included the treasuries holding each token in the API for additional verification if needed. Let me know if any further change is required, thanks.